### ISSUE TITLE
Add redshift like functionality

### ISSUE THREAD

**Issue by wolfv** [ErwanDouaille/gse-shader#7]
I think it would be perfect if Redshift worked again under Wayland. Maybe one could write a shader to implement the color shifting?

https://github.com/jonls/redshift

**Comment by ErwanDouaille** [comment]
Ok, what is Redshift ? A color temperature modifier ?

**Comment by wolfv** [comment]
Yes! To use in the nighttime. It's supposed to be good for the eyes and for
sleep.
